glide.lock

```diff
@@ -11,13 +11,43 @@ imports:
   - autorest/validation
 - name: github.com/dgrijalva/jwt-go
   version: 24c63f56522a87ec5339cc3567883f1039378fdb
+- name: github.com/howeyc/gopass
```

Comment:
just curious, what are the reasons of adding more dependencies on packages? Are these for development or production usage?